services/core-api/app/api/dams/models/dam.py

```diff
@@ -15,14 +15,14 @@ class ConsequenceClassification(Enum):
     NRT = 'NRT'
 
     def __str__(self):
-        return self.value
+        return self.value[0]
```

Comment:
this was interesting! I kept getting errors in the bit that converted these values to strings. As it turned out, `self.value` actually returned a Tuple. So it looked like `[ValueIWant,]`.